doc/api/python.rst.jinja

```diff
@@ -288,3 +288,80 @@ All item types recognize the following optional keys:
 
 {% endif %}
 {% endfor %}
+
+
+*********************
+Module ``eos.figure``
+*********************
+
+EOS also provides a new plotting framework with a different interface, which will replace
+the existing framework as soon as it is feature complete.
+
+.. note::
+
+   Import ``eos.figure`` before you do something like ``import matplotlib.pyplot as plt``,
+   because the ``eos.figure`` module sets its default plot style and a matplotlib backend.
+   All options (except the backend) can be overwritten by updating ``matplotlib.rcParams[...]``;
+   see also the ``matplotlib`` documentation.
+   Note that the default settings use LaTeX to create labels and math expressions,
+   so for this to work latex needs to be available on your system.
+
+
+Figures
+=======
+
+.. automodule:: eos.figure
+   :members:
+
+
+Plot types
+----------
+
+Below we list the available plot types.
+Each key `plot_type` corresponds to a class which is instantiated in the background.
+The user can supply keyword arguments which are documented under `eos.figure.plot`.
+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * - `plot_type`
+     - class
+     - description
+   {% for key, class, description in figure_plot_types %}
+   * - ``{{ key }}``
```

Comment:
double back ticks vs single back ticks

Reply:
Had another look and fixed the single back ticks. Single back ticks should only be used as arguments to a Sphinx or RST directive, e.g.,
```RST
This is an example linking to the Python class :py:class:`eos.figure.Figure`. It uses, amongst others,
the ``matplotlib.pyplot`` module.
```
Mentions of classes not included in our API documentation are using double back ticks to indicate code.